qa/TL1_jupyter_conda/test_cupy.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+[#!/bin/bash -e
```

Comment:
Out of curiosity, why the `[]` brackets around this test?

Reply:
Not intended